homeassistant/components/emulated_hue/upnp.py

```diff
@@ -74,13 +74,14 @@ def __init__(
         # Note that the double newline at the end of
         # this string is required per the SSDP spec
         resp_template = f"""HTTP/1.1 200 OK
-CACHE-CONTROL: max-age=60
+HOST: 239.255.255.250:1900
```

Comment:
```suggestion
```

The host line doesn't look right

Reply:
I've removed this line, could you try again with your harmony hub?

As for the other changes in the upnp discovery response: The free@home sysap only works with the changes I made. It specifically looks for an `upnp:rootdevice`, so I hope that is not causing the issue with your harmony hub:

![image](https://user-images.githubusercontent.com/22068/80926299-b042cc80-8d96-11ea-8703-d24a439deee1.png)